src/rules/iddataweb-verification-workflow.js

```diff
@@ -0,0 +1,106 @@
+  
+//initialize app metadata
+  user.app_metadata = user.app_metadata || {};
+  user.app_metadata.iddataweb = user.app_metadata.iddataweb || {};
+  
+//if coming back from redirect - get token, make policy decision, and update user metadata.
+    if (context.protocol === "redirect-callback") {
+        console.log("code from IDW: " + context.request.query.code);	
+      
+      let options = { method: 'POST',
+  url: configuration.idwBaseURL + '/axn/oauth2/token',
+  headers: 
+   { 'Cache-Control': 'no-cache',
+     Authorization: 'Basic ' + Buffer.from(configuration.idwClientID + ':' + configuration.idwClientSecret).toString('base64'),
+     'Content-Type': 'application/x-www-form-urlencoded' },
+  form: 
+   { grant_type: 'authorization_code',
+     code: context.request.query.code,
+     redirect_uri: 'https://' + context.request.hostname + '/continue'
+      } };
+
+      request(options, function (error, response, body) {
+        if (error) throw new Error(error);
+
+        let jsonData = JSON.parse(body);
+        let decodedToken = jwt.decode(jsonData.id_token);
+        console.log('policy decision: ' + decodedToken.policyDecision);
+        console.log('score: ' + decodedToken.idwTrustScore);
+        if (configuration.logIdwJwt === 'on'){
+          console.log(JSON.stringify(decodedToken)); 
+        }
+        
+        //if IDW verification was successful - update user's metadata in Auth0.
+        //this could be used for downstream application authorization,
+        //or mapping access to levels of assurance.
+        if (decodedToken.policyDecision === 'approve'){
+           user.app_metadata.iddataweb.verificationResult = 'verified';
+           auth0.users.updateAppMetadata(user.user_id, user.app_metadata)
+          .then(function(){
+            callback(null, user, context);
+          })
+          .catch(function(err){
+            callback(err);
+          });
+        } 
+      });
+
+      	return callback(null, user, context);
+      
+    } else {
+      
+      //if coming in for first time - redirect to IDW for verification.
+      if ((user.app_metadata.iddataweb.verificationResult === 'verified') && (configuration.alwaysVerify === 'off')){
+        console.log('user ' + user.user_id + ' has been previously verified.');
+        return callback(null, user, context);
+        //if not previously verified, redirect
+      } else {
+        //if "prefill attributes" is on, build and sign JWT, and include in /auth request to ID DataWeb.
+        if (configuration.prefillAttributes === 'on') {
+          
+          let prefillToken = jwt.sign(
+    {
+
+        "sub": user.email,
+        "credential": user.email,
+        "email": user.email,
+        "fname": user.given_name,
+        "lname": user.family_name,
+        "phone": user.phone_number
+
+    }, 
+    configuration.idwClientSecret, { expiresIn: '1h' });
+          
+          console.log(configuration.idwClientSecret);
```

Comment:
```suggestion
          console.log(configuration.IDDATAWEB_CLIENT_SECRET);
```

... but we probably don't want to log this.